### DISCUSSION TITLE
Are the 'four quadrants' from wilber applicable to groups/organizations?

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#959] [Research]
I haven't read that much from Wilber, but I'm curious if the four quadrants are ever applied from the level of the 'group'/an 'organization'? I ask in the sense that the quadrants imply an individual human being and how they 'fit into the world' (it seems), and I'm curious if this has been applied to how a group or organization 'fits into the world'. The tricky part being the top left quadrant of subjectivity (maybe there are more tricky parts) but it seems like it would be possible to consider how a particular group inspires certain feelings within each member. If it has been applied to orgs, what did they say about it? @asimong @rufuspollock

**Comment by asimong** [comment]
An astute question, thanks @matthewmccarthy11 . What comes up for me is the surprising lack of nuance in the binary distinction between self and group. There are several levels to group life, and they don't appear to be simply bigger = more complex and more advanced or developed. So, to me, it isn't actually quadrants, but two dimensions at least one of which has a scale. I'm reminded of my own writings on Collective Ikigai, where I have individual – group – world. Might be fertile ground to explore.

**Reply by catherinet1** [reply]
"There are several levels to group life" -- this reminds me of Microsolidarity and the different levels/sizes/scales of group life that they frame, e.g. see https://www.microsolidarity.cc/essays/five-scales-of-microsolidarity

**Reply by asimong** [reply]
Indeed @catherinet1 the microsolidarity concept set was in my mind. So, yes, worth bringing in that kind of framework.

**Comment by matthewmccarthy11** [comment]
Here is one such diagram: ![group-stage-theory](https://github.com/life-itself/community/assets/131390186/4afcec80-a547-4ee8-a539-8aa5b6e5e36c)

**Reply by asimong** [reply]
I've replied in #963

**Reply by asimong** [reply]
and to me it doesn't actually address the issue of individual/group naive individualism as an underlying mindset. See the reply on #963

**Reply by catherinet1** [reply]
Where is this diagram from @matthewmccarthy11 and what is it intending to show overall? e.g. what do the coloured rings inside the big circle represent? Ah - does each coloured ring represent the manifestation of each of the outside elements at 'red', 'orange' etc. levels?

**Reply by asimong** [reply]
As @matthewmccarthy11 noted on the other thread, he picked it up from https://twitter.com/NoraBateson/status/1425142926254956550 where Rich Bartlett had posted it without citing a source. And yes, I interpreted it the same way, with the "green" and "teal" outer rings being most obvious. As there isn't anything outside teal, I guess it might be from Laloux or his line of thinking.

**Comment by rufuspollock** [comment]
There's a whole long discussion in Wilber about social holons (i.e. groups) vs individual holons (e.g. people). the rough sense of it is that no, you don't apply the same ideas about dimensions of development to social holons as you do to individual holons.

I had long discussions with Doshin Roshi about this.
